.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules/
 daprdocs/public
 daprdocs/resources/_gen
 .venv/
+.hugo_build.lock
```

Comment:
Please remove this file. This is not intentional correct?

Reply:
it is